modules/engage-theodul-core/src/main/resources/ui/js/engage/models/meInfo.js

```diff
@@ -69,113 +68,45 @@ define(['jquery', 'backbone'], function($, Backbone) {
                         matomo_heartbeat,
                         matomo_notification,
                         matomo_track_events;
```

Comment:
The defaults are now ignored. E.g. when `layout` is not set in the YAML will is still be `"off"` as written above? From what I understand, it will be `undefined`.

Reply:
Yes, it would be undefined, Should be left with a default setting?